functions/src/emails.ts

```diff
@@ -390,3 +391,53 @@ export const sendNewAnswerEmail = async (
     { from }
   )
 }
+
+export const sendThreeContractsEmail = async (
+  privateUser: PrivateUser,
+  contractsToSend: Contract[]
+) => {
+  const emailType = 'weekly-trending'
+  const unsubscribeUrl = `${UNSUBSCRIBE_ENDPOINT}?id=${privateUser.id}&type=${emailType}`
+  if (!privateUser || !privateUser.email) return
+  await sendTemplateEmail(
+    privateUser.email,
+    contractsToSend[0].question + ' and 2 more questions for you.',
```

Comment:
Not sure about this as an email subject. Something like "Interesting markets this week" would be better, I think.

Reply:
If they're truly interesting then one of the titles should get their attention, right?

Reply:
I like Ian's idea to include the question.

What do you think about sending more markets? Like 6?

Reply:
hmmm!! yeah we might as well, for all we know these could be the only markets they see all week